docs/community/faq/faq-users.md

```diff
@@ -2,69 +2,85 @@
 
-2. What kind of data can I analyze with AutoEmulate?
+2. What kind of data does `AutoEmulate` need to build an emulator?
    <!-- Clarification on the types of datasets suitable for analysis, including data formats and recommended data sizes. -->
 
-3. How do I interpret the results from AutoEmulate?
+   - `AutoEmulate` takes simulation inputs `X` and simulation outputs `y` to build an emulator.`X` is an ndarray of shape `(n_samples, n_parameters)` and `y` is an ndarray of shape `(n_samples, n_outputs)`. Each sample here is a simulation run, so each row of `X` corresponds to a set of input parameters and each row of `y` corresponds to the corresponding simulation output. Currently, all inputs and outputs should be numeric, and we don't support missing data.
+
+   - All models work with multi-output data. We have optimised `AutoEmulate` to work with smaller datasets (in the order of hundreds to thousands of samples). Training emulators with large datasets (hundreds of thousands of samples) may currently require a long time and is not recommended.
+
+3. How do I interpret the results from `AutoEmulate`?
    <!-- Guidance on understanding the output of the software, including any metrics or visualizations it produces. -->
+   - See the [tutorial](../../tutorials/01_start.ipynb) for an example of how to interpret the results from `AutoEmulate`. Briefly, `X` and `y` are first split into training and test sets. Cross-validation and/or hyperparameter optimisation are performed on the training data. After comparing the results from different emulators, the user can evaluate the chosen emulator on the test set with `autoemulate.evaluate_model()`, and plot test set predictions with `autoemulate.plot_model()`. 
+
+   - An important thing to note is that the emulator can only be as good as the data it was trained on. Therefore, the experimental design (on which points the simulation was evaluated) is key to obtaining a good emulator.
 
-4. Can I use AutoEmulate for commercial purposes?
+4. Can I use `AutoEmulate` for commercial purposes?
    <!-- Information on licensing and any restrictions on use. -->
+   - Yes. It's licensed under the MIT license, which allows for commercial use. See the [license](../../../LICENSE) for more information.
 
 ## Advanced Usage
 
-1. How can I customize simulations in AutoEmulate?
-   <!-- Explanation of how users can adjust parameters or settings to tailor simulations to their specific research questions. -->
-
-2. Does AutoEmulate support parallel processing or high-performance computing (HPC) environments?
+1. Does AutoEmulate support parallel processing or high-performance computing (HPC) environments?
    <!-- Details on the software's capabilities to leverage multi-threading, distributed computing, or HPC resources to speed up computations. -->
+   - Yes, `setup()` has an `n_jobs` parameter which allows to parallelise cross-validation and hyperparameter optimisation.
```

Comment:
Let's link `setup()` to the API docs like above!

Reply:
Linking now to the `autoemulate.compare` module, as I'm currently not sure how to link to a method within the module. I think this needs modification of the rst file, but i don't know whether this should be done manually.